soc/arm/nxp_imx/rt/soc_rt11xx.c

```diff
@@ -334,6 +334,11 @@ static ALWAYS_INLINE void clock_init(void)
 	rootCfg.mux = kCLOCK_BUS_ClockRoot_MuxSysPll3Out;
 	rootCfg.div = 2;
 	CLOCK_SetRootClock(kCLOCK_Root_Bus, &rootCfg);
+#if defined(CONFIG_COUNTER_MCUX_PIT)
+	rootCfg.mux = kCLOCK_BUS_LPSR_ClockRoot_MuxSysPll3Out;
+	rootCfg.div = 2;
+	CLOCK_SetRootClock(kCLOCK_Root_Bus_Lpsr, &rootCfg);
```

Comment:
I am not sure it is appropriate to do this code based on whether CONFIG_COUNTER_MCUX_PIT is defined, since it changes the configuration of the LPSR root, which is for all of the low power peripherals in the chip

Reply:
I am open to an alternative,
At the moment, 
IF the device is an 1166_CM7 or an 1176_CM7
then this is the only spot where the LPSR is configured,
and is the first time LPSR gets configured which means it can be
overwritten by the more important peripheral clocking setups.
Another thing to consider is that the PIT2 can only be rooted from the LPSR Root.

Reply:
I think we just don't need to guard it with #if PIT

Reply:
Check this again, I put this in a more convenient spot.

Reply:
My comment was not about code format, it was a functional concern with conditional compilation paths

Reply:
The LPSR Clock root is no longer conditioned by the PIT Kconfig